app/views/members/_search.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_tag(members_path, method: :get) do %>
+  <%= text_field_tag :query, params[:query] %>
+  <%= submit_tag 'Search', name: nil %>
```

Comment:
If the goal is preventing the `commit=Search` from showing up in the URL, this is probably the most straightforward way to do that, especially since the button value is not of importance to the form submission. That context was not immediately clear, hence my question :)